zeg/datasets.py

```diff
@@ -67,13 +74,29 @@ def update(log, session, args):
     log.debug("File mime: {}".format(file_mime))
 
     with open(file_path) as f:
-        response_json = http.post_file(
-            session,
-            url,
-            file_name,
-            f,
-            file_mime)
-        log.print_json(response_json, "dataset", "update")
+        blob_id = str(uuid.uuid4())
+        info = {
+            "image": {
+                "blob_id": blob_id,
+                "name": file_name,
+                "size": os.path.getsize(file_path),
+                "mimetype": file_mime
+            }
+        }
+        create = http.post_json(session, url_url, info)
+        # Post file to storage location
+        url = create["url"]
+        if url.startswith("/"):
+            url = 'https://storage.googleapis.com{}'.format(url)
+        http.put_file(session, url, f, file_mime)
+        # confirm
+        current = http.get(session, url)["dataset"]
```

Comment:
This doesn't work: In both my e2e tests, and when running manually, I get an error from this get.
> Unhandled exception: 403 b"<?xml version='1.0' encoding='UTF-8'?><Error><Code>SignatureDoesNotMatch</Code><Message>The request signature we calculated does not match the signature you provided. Check your Google secret key and signing method.</Message><StringToSign>GET\n\n\n1575390112\n/staging-6jjggstq/1771c59a-8fa9-4dbf-98ce-b5e9461900c4</StringToSign></Error>"

Are the signed urls single use only or something?